src/components/ItaliaTheme/Blocks/TextCard/CardWithImage/Block.jsx

```diff
@@ -35,7 +35,7 @@ const renderImage = (image, showImage, sizeNatural) =>
         <figure className="img-wrapper">
           <img
             src={`data:${image['content-type']};${image.encoding},${image.data}`}
-            alt=""
+            alt={altText || ''}
```

Comment:
```suggestion
            alt={altText}
```

Togli il default da qui e metti sopra

Reply:
Fatto!